### PR TITLE
Create install.md

### PR DESCRIPTION
**Pull request by dwbrown2**
Draft install steps for OpenCost website.

@AjayTripathy can you please review? I'm not sure if other steps are needed to create a new markdown file here.

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+* You can run [helm template](https://helm.sh/docs/helm/helm_template/) against the [Kubecost helm chart](https://github.com/kubecost/cost-analyzer-helm-chart/) to generate local YAML output. This requires extra effort when compared to directly installing the helm chart but is more flexible than deploying static YAML.
+
+* You can install via [flat manifest](https://github.com/kubecost/cost-analyzer-helm-chart/blob/master/README.md#manifest). This install path provides less flexibility for managing your deployment and has several product limitations, e.g. Thanos is not easily enabled.
+
+* Lastly, you can deploy the OpenCost directly as a pod. Directions for this install path are available [here](https://github.com/kubecost/opencost/blob/master/deploying-as-a-pod.md). This install path just deploys the underlying OpenCost allocation model with a limited UI or is not meant to be enterprise-ready.
+
+Edit this doc on [GitHub](https://github.com/kubecost/docs/blob/main/install.md)
+
+<!--- {"article":"4407601821207","section":"4402815636375","permissiongroup":"1500001277122"} --->
```
Comment (AjayTripathy) [inline docs/install.md:27]: I think this breaks the build and you need to delete this line.

Comment (dwbrown2) [reply docs/install.md]: Deleted but still see a build failure. Any suggestions?

**Comment by AdamStack18** [comment]
@AjayTripathy, Could you help Webb with the build failure when you have time?

**Comment by elenalape** [comment]
fixed! there was an incopatible html tag, `<ins>` (TIL it exists).

**Pull request merged**